x-pack/plugins/index_management/server/config.ts

```diff
@@ -52,6 +52,11 @@ const schemaLatest = schema.object(
       // We take this approach in order to have a central place (serverless.yml) for serverless config across Kibana
       serverless: schema.boolean({ defaultValue: true }),
     }),
+    enableMappingsSourceField: offeringBasedSchema({
```

Comment:
I was thinking about renaming the config to `enableMappingsSourceFieldConfiguration` but this might be too long. I'm open to any suggestions for the name of the config.

Reply:
Thanks for the suggestions! I like `enableMappingsSourceFieldSection` since the component for the _source field configuration is called `SourceFieldSection`. Added this change with [813297e](https://github.com/elastic/kibana/pull/181712/commits/813297eeb5a3c5c35befec9b4b3ddd912085e129).